gsw_xarray/_util.py

```diff
@@ -31,3 +41,17 @@ def submodule_all_compat(submodule_name):
         all = dir(submodule)
         # as per python docs, an import * should return everything not starting with "_"
         return list(filter(lambda s: not s.startswith("_"), all))
+
+
+def get_attribute(name):
+    if name in _compat:
+        return getattr(_compat_modules[name], name)
```

Comment:
The name `_compat_modules` does not exist in this context, the only way I figured out how to trigger this is by trying to get one of the modules from the accessor:
```python
ds.gsw["conversions"]
```
Which doesn't really make sense since it is not a teos-10 property to calculate. There might be some way to trigger this from someplace else. Though all the tests pass so not sure.

Reply:
Well caught. I would be in favor of not providing access to modules in the accessor, as is it not the recommended way to do.
We keep the modules for drop in replacement (using `import gsw` or `import gsw_xarray as gsw` are equivalent), but remove them from accessor. Is it fine for you?

Reply:
No that I test it I realized that I had not understood your comment. I dive into the code to find a solution

Reply:
Resolved by [ba635d5](https://github.com/DocOtak/gsw-xarray/pull/53/commits/ba635d55efdc5ba8890c2777bd7976a64eea1abe)